### DISCUSSION TITLE
paid content

### DISCUSSION THREAD

**Discussion by GabanKillasta** [Avnsx/fansly#35] [Q&A] [answered]
is there a way of making it download paid content that hasent been paid for already? or is it supposed to already, if so then it isnt for me

**Comment by Avnsx** [comment, marked as answer]
No

**Comment by olivechicago** [comment]


**Reply by Avnsx** [reply]
You can't get the API backend url's for paid content, unless you've actually paid ...

**Reply by olivechicago** [reply]
lol, my fault.  Somehow I overlooked the "hasn't" part of his question.

My suggestion was not incorrect though, as I did say "to DL _paid_ content."
